src/components/Book.js

```diff
@@ -1,24 +1,22 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import Books from './Books';
-
-const BookContent = ({ books }) => (
-  <ul className="book-content">
-    {
-books.map((book) => <Books key={book.id} book={book} />)
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/books';
+/* eslint-disable react/prop-types */
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.